microsite/data/plugins/microsoft-calendar.yaml

```
@@ -0,0 +1,16 @@
+title: Microsoft Calendar
+author: StatusNeo
+authorUrl: https://statusneo.com/
+category: Productivity
+description: |
+  The plugin integrates with your Microsoft Calendar.
+  It displays all your calendars and events in it.
+documentation: https://statusneo.com/23392-2
```

```suggestion
documentation: https://github.com/backstage/backstage/tree/master/plugins/microsoft-calendar
```
Let's reference the actual repository